src/network/MessageContainer.cpp

```diff
@@ -4,6 +4,7 @@
 
 #include "MessageContainer.hpp"
 #include <util/OptionalSerialization.hpp>
+#include <time.h>
```

Comment:
```suggestion
#include <ctime>
```
`time.h` ist deprecated